apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -5,35 +5,76 @@ import {
 	type FieldValues,
 } from "react-hook-form";
 
-import { useFormController } from "~/libs/hooks/hooks.js";
+import { useCallback, useFormController } from "~/libs/hooks/hooks.js";
 
 type Properties<T extends FieldValues> = {
+	className: string;
```

Comment:
for all Input component calls inside of sign-up-form you provide a single value to the className prop, "input-container". just put this value as a classname inside of component's JSX instead of passing the same prop every time you use this component

Reply:
agree with you, i think this make much better